frontend/src/modules/conversation/components/conversation-dropdown.vue

```diff
@@ -21,24 +21,6 @@
         ><i class="ri-link mr-1" />Copy Public
         Url</el-dropdown-item
       >
-      <el-dropdown-item
-        v-if="!conversation.published"
-        :command="{
-          action: 'conversationPublish',
-          conversation: conversation
-        }"
```

Comment:
I'm not sure if we can remove these, I'm using them on the Community Help Center list table
Can we figure out another way to not display them in the context you need?